app/setup_handlers.go

```diff
@@ -30,7 +30,7 @@ func loadUpgradeStore(app *ElysApp) {
 
 	if shouldLoadUpgradeStore(app, upgradeInfo) {
 		storeUpgrades := storetypes.StoreUpgrades{
-			// Added: []string{},
+			Added: []string{"incentive"},
```

Comment:
I am not sure why this upgrade handler stuff ("incentive") puit on tendermint -> cometbft upgrade

Reply:
it is because it was missing on v0.3.1, when we created the new module incentive we need to create the module kv store the first time, it is part of the migration init process

Reply:
Okay